lib/automate_liveness_agent/config.rb

```diff
@@ -149,6 +151,10 @@ def apply_config_values(config_data)
         sanity_check_daemon_mode(config_data["daemon_mode"])
       end
 
+      if config_data.key?("interval")
+        @interval = config_data["interval"]
```

Comment:
Can we add a sanity check to assert that this is either `nil` or a string of digit chars? 

```
"lkjlkj".to_i
=> 0
```
... that would probably not be good